src/settings/Settings.tsx

```diff
@@ -220,6 +221,26 @@ export const SETTINGS: { [setting: string]: ISetting } = {
             requiresRefresh: true,
         },
     },
+    [Features.NotificationSettings2]: {
+        isFeature: true,
+        labsGroup: LabGroup.Experimental,
+        supportedLevels: LEVELS_FEATURE,
+        displayName: _td("New Notification Settings"),
+        default: false,
+        betaInfo: {
+            title: _td("Notification Settings"),
+            caption: () => (
+                <>
+                    <p>
+                        {_t("A simpler way to customize notification settings in %(brand)s just the way you like.", {
+                            brand: SdkConfig.get().brand,
+                        })}
+                    </p>
+                </>
+            ),
+            image: require("../../res/img/betas/notification_settings.png"),
```

Comment:
The image used here is very text heavy and only available in English, I don't think this is appropriate for non-English user-base.

Reply:
this is for the Labs flag? 
it's relevant push back but we're only planning on being in labs for one cycle so unless there's a straight-forward alternative it's not something i think we should put time to right now

Reply:
Its an accessibility issue, not dissimilar to using gifs here

Reply:
What's the alternative?

Reply:
Not having an image

![image](https://github.com/matrix-org/matrix-react-sdk/assets/2403652/8bce5b9c-1740-468e-9c1e-b34019415d4e)